packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -107,7 +107,6 @@ export function Comments( {
 					}
 				}
 			} );
-			setSelectedThread( 'new-note-thread' );
```

Comment:
I think this wasn't doing anything. The new note form is always selected when visible.